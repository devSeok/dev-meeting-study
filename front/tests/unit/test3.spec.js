import repeat from "./jestTest/snapShot";
describe("group6", () => {
  // test("repeats words three times", () => {
  //   expect(repeat("Test", 3)).toMatchInlineSnapshot(`"Test,Test,Test"`);
  // });

  test("repeats words three times", () => {
    expect(repeat("Test", 3)).toMatchSnapshot();
  });
});
