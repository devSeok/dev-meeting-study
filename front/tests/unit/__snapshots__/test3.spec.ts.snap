// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`group6 repeats words three times 1`] = `
Array [
  "Test",
  "Test",
  "Test",
]
`;
